client/src/pages/Courses/RegulationModalValidation.js

```diff
@@ -0,0 +1,345 @@
+// import React, { useState, useEffect } from "react";
+// import {
+//     FormControl,
+//     Input,
+//     Modal,
+//     ModalOverlay,
+//     ModalContent,
+//     ModalHeader,
+//     ModalFooter,
+//     ModalBody,
+//     ModalCloseButton,
+//     Button,
+//     Stack,
+//     TagLabel,
+//     Tag,
+//     TagCloseButton,
+//     NumberInput,
+//     NumberInputField,
+//     NumberInputStepper,
+//     NumberIncrementStepper,
+//     NumberDecrementStepper,
+//     Flex,
+//     IconButton,
+//     Text,
+//     Switch,
+//     FormLabel,
+//     ButtonGroup,
+// } from "@chakra-ui/core";
+
+// const defaultReg = {
+//     type: "POINTS",
+//     points: 15,
+//     courses: [],
+// };
```

Comment:
Should the file be deleted?

Reply:
Nah, we want to keep it in case we decide to go do validation properly again

Reply:
Hm... I feel we shouldn't commit an unused file for the sake of keeping things clean. Should we make a gist of this and save it on discord if we ever need it in the future?